pytest_odoo.py

```diff
@@ -237,7 +237,9 @@ def _find_manifest_path(collection_path: Path) -> Path:
     while level < 5  and not (path.parent / "__manifest__.py").is_file():
         path = path.parent
         level += 1
-    return path.parent / "__manifest__.py"
+    if (path.parent / "__manifest__.py").is_file():
```

Comment:
Hum I think if collection_path already contain the manifest it should return it self. reading your code I think this is not the case. I've fixed that in a closed (pr #70) with unit-test !  

I'll pick my commits and create a new PR with that fix independantly

Reply:
hum... the case would be, if someone add test_ file at the same level as __manifest__ I suppose!

Reply:
here you are https://github.com/camptocamp/pytest-odoo/pull/73